src/file/file.service.ts

```diff
@@ -178,8 +175,7 @@ export class FileService {
     let files: IFile[];
     if (!folderID) { // Search the user's root folder
       if (!ownerID) throw new ClientError('No file or owner id sent');
```

Comment:
This if should be at the higher level, there cannot be a file that has no owner and you cannot query for the files without it.